main.py

```diff
@@ -1,10 +1,20 @@
-import cv2
+import cv2, face_recognition
 #get webcam
```

Comment:
a [doc string](https://peps.python.org/pep-0257/) for the file as a whole would be good. What's the point of this script? What are you going to achieve with it?